variables.tf

```diff
@@ -56,6 +57,7 @@ variable "plan" {
   validation {
     condition = anytrue([
       var.plan == "enterprise",
+      var.plan == "platinum",
     ])
     error_message = "Only supported plan is enterprise"
```

Comment:
This error message needs to be updated. Perhaps also add supported values to the variable description